integration_tests/e2e/prePrisonEducaton/createPrePrisonEducation.cy.ts

```diff
@@ -104,7 +104,7 @@ context('Create a prisoners pre-prison education', () => {
     const overviewPage = Page.verifyOnPage(OverviewPage)
     overviewPage.selectTab('Education and training')
     const educationAndTrainingPage = Page.verifyOnPage(EducationAndTrainingPage)
-    educationAndTrainingPage.clickToAddEducationalQualifications(HighestLevelOfEducationPage)
+    educationAndTrainingPage.clickToAddEducationHistory()
```

Comment:
I've had to rename (and slightly change the semantics) of the existing method `clickToAddEducationalQualifications` because that method name makes more sense in the way I want to use it for this bug fix.
The new name `clickToAddEducationHistory` is a better name for the use case of _this_ cypress test